src/main/webapp/WEB-INF/view/chat.jsp

```diff
@@ -44,12 +44,27 @@ List<Message> messages = (List<Message>) request.getAttribute("messages");
     };
   </script>
 </head>
+
 <body onload="scrollChat()">
 
   <nav>
     <%@include file="navbar.jsp"%>
   </nav>
 
+<!-- This is the test UI for the pop up box -->
+    <button onclick="getMessageId()">Try it</button>
+    <script>
```

Comment:
can you move all javascript to bottom of file? Rather than right after the invocation. Generally it should be

//layout stuff//

<script>
//all js
</script>